app/views/layouts/application.html.haml

```diff
@@ -14,7 +14,7 @@
     %meta{:name => "author", :content => "Diaspora, Inc."}/
 
     %link{:rel => 'shortcut icon', :href => "#{image_path('favicon.png')}" }
-    %link{:rel => 'apple-touch-icon', :href => "#{image_path('apple-touch-icon.png')}"}
+    %link{:rel => 'apple-touch-icon', :href => "#{image_path('favicon.png')}"}
```

Comment:
> In any case, I suspect there are lots of images that aren't used anymore in the interface and that could be suppressed.
@AugierLe42e: i started cleanup the unused images yesterday: https://github.com/SuperTux88/diaspora/commits/cleanup-assets ... so you don't have to cleanup any images in this PR